website/docs/migrations/migrate-to-v0.19.md

```diff
@@ -1,88 +1,77 @@
 # Migration to Lumos@0.19
 
-Migrate from lumos@0.18 or earlier to Lumos@0.19
+Migrate from `lumos@0.18` or earlier to `lumos@0.19`
 
 1. changed all snake_case to camelCase
-2. removed the ckb-js-toolkit and replaced with @lumos/codec
+2. deprecated the `@ckb-lumos/toolkit` and replaced with `@ckb-lumos/codec`
+3. deprecated `readBigUintX` in `@ckb-lumos/base`
 
-## At a Glance
+## Step-by-Step Guide
 
-Migration to the new version should contain changes similar to:
+### Step 1
 
-```diff
-// usage of snake_case types
-const lockScript: Script = {
--  code_hash: '0x...',
--  hash_type: "type",
-+  codeHash: '0x...',
-+  hashHype: "type",
-  args: '0x...',
-};
-
-// usage of snake_case RPC methods
--  const hash = await rpc.send_transaction(tx, "passthrough");
-+  const hash = await rpc.sendTransaction(tx, "passthrough");
+Download the migration script and run it to automatically change your snake-case code to camel-case. Before doing this, you should commit your code to git and keep repo clean with no staging changes.
```

Comment:
maybe we should make the warning first to tell user running the code mode will change their code

<pre>
### Step 1 - Run the Codemod

**WARNING**: Running the codemod will change your source code, please make sure your code are has been committed

Running the codemod will

1. change your lumos-related API from snake_case to camelCase
2. warning for deprecated function

```sh
curl -s https://gist.githubusercontent.com/zhangyouxin/e5ddf9b966f611173a01d6c98715c931/raw/30606c12756a866437a390add8dc5f5df47c9c36/codemod.js \
| xargs -0 -I{} node -e {} "your-source-dir/**/*.js"
```
</pre>